src/app/Marine2/components/boxes/Battery/Battery.tsx

```diff
@@ -1,13 +1,29 @@
 import Box from "../../ui/Box"
 import { Battery as BatteryType } from "@elninotech/mfd-modules"
-import { batteryStateNameFormatter } from "../../../utils/formatters"
+import { batteryStateNameFormatter, colorForPercentageFormatter } from "../../../utils/formatters"
 import { BATTERY } from "../../../utils/constants"
 import BatteryChargingIcon from "../../../images/icons/battery-charging.svg"
 import BatteryIcon from "../../../images/icons/battery.svg"
 import BatteryDischargingIcon from "../../../images/icons/battery-discharging.svg"
-import { colorForPercentageFormatter } from "../../../utils/formatters"
 import { translate } from "react-i18nify"
 import classNames from "classnames"
+import { applyStyles, StylesType } from "app/Marine2/utils/media"
+import { useState } from "react"
+
+const styles: StylesType = {
+  "sm-s": {
+    mainValue: "text-2xl",
+    subValue: "text-base",
+  },
+  "md-s": {
+    mainValue: "text-3xl",
+    subValue: "text-base",
+  },
+  "lg-s": {
+    mainValue: "text-3xl",
+    subValue: "text-lg",
+  },
+}
```

Comment:
What about the default styles, which are applied when the box is smaller than `sm-s`? I think those should be right here as well, that way it is easy to see what the styles will be at which breakpoint instead of having to scroll back to the component(s) it is being applied to.

Reply:
That's a nice idea, will add it now